### DISCUSSION TITLE
Project Discussion

### DISCUSSION THREAD

**Discussion by Ishani-Chatterjee** [TEAMMATES/teammates#11332] [Q&A] [answered]
Hello, I am new in this community and want to start understanding and contributing in one of the project ideas. Can anyone please help me out and guide me through this?

**Comment by damithc** [comment, marked as answer]
@Ishani-Chatterjee this is a good place to start https://github.com/TEAMMATES/teammates/blob/master/docs/CONTRIBUTING.md
